quinn-proto/src/connection/streams.rs

```diff
@@ -52,8 +52,13 @@ pub struct Streams {
     connection_blocked: Vec<StreamId>,
     /// Connection-level flow control budget dictated by the peer
     max_data: u64,
-    /// Limit on incoming data
+    /// The initial receive window
+    receive_window: u64,
```

Comment:
This is now mainly required to be stored in order to know what is "significant".
In order to avoid it, we could pick a fixed size. Or we could likely go to u32 and save a few bytes if we store the minimum diff that is required to be transmitted. I'm good with any option.

Reply:
This seems fine to me as-is. I wouldn't mind the u32 option but I think this has fewer edge cases.